tests/fixtures.py

```diff
@@ -284,9 +284,19 @@ def enacted_blockchain_policy(idle_blockchain_policy, blockchain_ursulas):
 
 @pytest.fixture(scope="module")
 def capsule_side_channel(enacted_federated_policy):
-    enrico = Enrico(policy_encrypting_key=enacted_federated_policy.public_key)
-    message_kit, _signature = enrico.encrypt_message(b"Welcome to the flippering.")
-    return message_kit, enrico
+    class _CapsuleSideChannel:
```

Comment:
Another Side Channel... We are accumulating them!

Reply:
This is a replacement; we are net zero side channels here.  :-)